config/armada/config.yaml

```diff
@@ -17,60 +17,99 @@ grpc:
     permitWithoutStream: false
 redis:
   addrs:
-    - "localhost:6379"
+    - "redis:6379"
```

Comment:
I don't understand this- how does this work if you're running armada in an ide?  Or is this config no longer used for that?

Reply:
I doesn't, the default config is for running in containers.
To run in an IDE, this value needs to be overridden, e..g, as shown here
https://github.com/G-Research/armada/blob/master/docs/development_guide.md#running-the-armada-server-and-executor-in-visual-studio-code

Alternatively, we could move the overrides from env variables into a separate config file with only overrides.